ext/spl/spl_array.c

```diff
@@ -822,7 +822,7 @@ static HashTable *spl_array_get_properties_for(zval *object, zend_prop_purpose p
 	return ht;
 } /* }}} */
 
-static HashTable* spl_array_get_debug_info(zval *obj, int *is_temp) /* {{{ */
+static zend_always_inline HashTable* spl_array_get_debug_info(zval *obj, int *is_temp) /* {{{ */
```

Comment:
Don't think there's much value in always inlining the get_debug_info functions. They'll be inlined anyway in release builds (static function with single use), and there's no need to inline them in debug builds.

Reply:
Agreed!